### ISSUE TITLE
[A11y] Collection Landing - Text Zoom

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1832]
### Operating System

Windows

### Browser

Edge

### Affected page

Collection Show

### Admin user?

Yes

### Issue Description

At 200% zoom, the "VIEW ALL" button is clipping text, the search at the bottom of the page is clipping text, and the text that shows on hover for each of the collection's featured images is clipped.

At 400% zoom and 1280x1024, "VIEW ALL" and "SEARCH" are fine, but the text that shows on featured images' hover is gone entirely.

The further you zoom in, the smaller the featured images tend to get.  Their focus "border" becomes so thick that on focus you can't see the image anymore.

The buttons need to resize more gracefully.  At 150% zoom they're fine, but for some reason 175% zoom (in Edge) makes them break even though there would be plenty of space to keep the buttons big enough for their text.  Once you hit 300% zoom, they work again.

Once you get into the 300% zoom range, it's clipping vertically, too, and eventually is completely invisible because it's trying to display below the image.

Examples attached:

Normal zoom:

![image](https://user-images.githubusercontent.com/2157974/128735314-f14477d6-90d4-4921-a549-e93b1f171758.png)

175% zoom (note the text clipping on the button and the featured image):

![image](https://user-images.githubusercontent.com/2157974/128735439-8bf65f1c-b5b4-49b9-954c-f5068e83cfc8.png)

400% zoom, featured images (note lack of text and images getting smaller):

![image](https://user-images.githubusercontent.com/2157974/128735655-a39bf696-7720-4097-914e-0fac5d7c618a.png)

400% zoom, button:

![image](https://user-images.githubusercontent.com/2157974/128735702-de37db7f-98b5-44ef-893b-1c97a86022e2.png)

### Suggested Fix

The buttons need to be able to handle zoom better - there is plenty of space for them at every zoom level, so they just need to adapt better.

The text on the featured images is a problem for many reasons already mentioned; this is just another issue.  Actual captions that are always visible should replace the hover text.

The featured images shouldn't get smaller as a person is trying to zoom in.  They should probably stack instead of continuing to try and line up horizontally.

**Comment by jechols** [comment]
QA: zoom in at a variety of levels and screen sizes and verify there's no text clipping at any zoom level (up to 400%)

**Comment by CGillen** [comment]
Mostly fixed by #1915 
Additional work needed for featured images

**Comment by jechols** [comment]
@CGillen It looks to me like the issue with the buttons has been fixed. Do you want me to file a new ticket (or multiple tickets) with the remaining issues?

**Comment by jechols** [comment]
Sent back to ready, pending response. (I think some of the other problems already have other tickets, but I'm not sure if they **all** do)

**Comment by straleyb** [comment]
@jechols Yeah, we should create a new ticket(s) with the remaining issues. This seems to be mostly about the clipping of the buttons at the higher zooms.

**Comment by jechols** [comment]
I think the text clipping is fine. I'm not sure about the featured images due to them no longer being on the collection landing page... not sure what to make of that.

I'll raise that issue and decide what to do about this ticket when that's figured out....

**Comment by jechols** [comment]
All good!
